CONTRIBUTING.md

```diff
@@ -98,6 +98,16 @@ This path should prioritize:
    5. Contributing to Bevy itself.
 6. Linking to further reading: official examples, `docs.rs` and (very sparingly) source code links.
 
+## Setup
```

Comment:
I could use more feedback and suggestions on this: I'm not very familiar with it.

I'm also not convinced it belongs here. In a lot of ways, this feels like it belongs in the READMEs of the generate-* folders?

Reply:
Well, it's kinda part of the setup to run the site, so it should probably be in both the README and in here.

The only thing I would make sure to specify is to run the scripts directly in the folders because it won't work correctly if you run it from the top level folder.